src/styles/search-bar.css

```diff
@@ -0,0 +1,48 @@
+.search-bar {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
```

Comment:
It's recommended to have a fallback font-family in case the inherited font is not available. Consider adding a generic font family as a backup.